blocks-config/uagb-controls/generateBackgroundCSS.js

```diff
@@ -23,6 +23,10 @@ function generateBackgroundCSS ( backgroundAttributes ) {
     } = backgroundAttributes;
 
     const bgCSS = {};
+    const xPositionValue = ( isNaN( xPosition ) ) ? 0 : xPosition;
```

Comment:
isNaN returns false for '' empty string
should we add condition as 
```suggestion
    const xPositionValue = ( isNaN( xPosition ) || '' === xPosition ) ? 0 : xPosition;
```